plugins/image-prioritizer/class-image-prioritizer-img-tag-visitor.php

```diff
@@ -90,17 +90,43 @@ public function __invoke( OD_Tag_Visitor_Context $context ): bool {
 		if ( is_null( $element_max_intersection_ratio ) ) {
 			$processor->set_meta_attribute( 'unknown-tag', true ); // Mostly useful for debugging why an IMG isn't optimized.
 		} else {
-			// Otherwise, make sure visible elements omit the loading attribute, and hidden elements include loading=lazy.
+			// TODO: Take into account whether the element has the computed style of visibility:hidden, in such case it should also get fetchpriority=low.
 			$is_visible = $element_max_intersection_ratio > 0.0;
-			$loading    = $get_attribute_value( 'loading' );
-			if ( $is_visible && 'lazy' === $loading ) {
-				$processor->remove_attribute( 'loading' );
-			} elseif ( ! $is_visible && 'lazy' !== $loading ) {
+			if ( true === $context->url_metric_group_collection->is_element_positioned_in_any_initial_viewport( $xpath ) ) {
```

Comment:
No need to do a `method_exists()` check here because Image Prioritizer here now requires Optimization Detective version 0.7.0+, which is the version this method is introduced in.